src/checkers/inference/InferrableAnnotatedTypeFactory.java

```diff
@@ -0,0 +1,32 @@
+package checkers.inference;
+
+import org.checkerframework.framework.type.treeannotator.TreeAnnotator;
+
+/**
+ * Interface for the annotatedTypeFactory that wish to let
+ * InferenceAnnotatedTypeFactory use the TreeAnnotator from current type system.
+ */
+
+public interface InferrableAnnotatedTypeFactory {
+
+    /**
+     * The default TreeAnnotator used by InferenceAnnotatedTypeFactory is
```

Comment:
Try to reword this to make it's purpose clearer. There is no default here, so you need to refer to what you mean.
Start with when this method would be implemented and then talk about what happens if that doesn't happen.